### ISSUE TITLE
Convert vg to gfa format terminate 

### ISSUE THREAD

**Issue by Chipcui** [vgteam/vg#2894]
Hi,
When I run the following commands to convert vg to gfa format, it terminated like this :
Commands:
```
vg construct -r westar.fa -v FAFU-ZS11.all.sort.filter.vcf >FAFU-ZS11.westar.vg
vg view FAFU-ZS11.westar.vg >FAFU-ZS11.westar.vg.gfa
```
Error:
```
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
ERROR: Signal 11 occurred. VG has crashed. Run 'vg bugs --new' to report a bug.
Stack trace path: /tmp/vg_crash_h505aj/stacktrace.txt
Please include the stack trace file in your bug report!
``` 
```
/tmp/vg_crash_h505aj/stacktrace.txt
Crash report for vg v1.25.0 "Apice"
Stack trace (most recent call last):
#24   Object "", at 0x4dfb19, in
#23   Object "", at 0x1c81848, in
#22   Object "", at 0x40a137, in
#21   Object "", at 0xa2a8f7, in
#20   Object "", at 0xa391b0, in
#19   Object "", at 0xdfef39, in
#18   Object "", at 0xe88056, in
#17   Object "", at 0xe87f60, in
#16   Object "", at 0xfee3b1, in
#15   Object "", at 0x1b8e8ad, in
#14   Object "", at 0x1c2fbd1, in
#13   Object "", at 0x1bc9aa3, in
#12   Object "", at 0x1bcbb00, in
#11   Object "", at 0x1bcbab5, in
#10   Object "", at 0x1c63e94, in
#9    Object "", at 0x1c92110, in
#8    Object "", at 0x1240287, in
#7    Object "", at 0x12402df, in
#6    Object "", at 0xf202bb, in
#5    Object "", at 0x1929ce1, in
#4    Object "", at 0x192a929, in
#3    Object "", at 0x192a68f, in
#2    Object "", at 0x193607d, in
#1    Object "", at 0x19350b5, in
#0    Object "", at 0x1932cc3, in
```
So, how to solve this error?
Thank you in advance!

**Comment by jeizenga** [comment]
`bad_alloc` is usually caused by running out of memory. Sometimes it's a bug in the code that asks for the memory, but it can also be because the system doesn't have sufficient resources. What kind of environment are you running vg in?

**Comment by glennhickey** [comment]
vg view still converts to protobuf internally when writing gfa, which takes
tons of memory.

**Comment by glennhickey** [comment]
This was fixed in #3120
